surfman/Cargo.toml

```diff
@@ -68,6 +68,17 @@ mach = "0.3"
 metal = "0.18"
 objc = "0.2"
 
+[target.aarch64-apple-darwin.dependencies]
```

Comment:
Rather than duplicating this list, I recommend using `[target.'cfg(target_os = "macos")'.dependencies]` instead.